src/components/v5/common/ActionSidebar/partials/forms/SplitPaymentForm/partials/SplitPaymentPercentField/SplitPaymentPercentField.tsx

```diff
@@ -26,7 +26,8 @@ const SplitPaymentPercentField: FC<SplitPaymentPercentFieldProps> = ({
         }
       }}
       wrapperClassName="flex-row flex text-md"
-      type="number"
+      type="text"
+      shouldAllowNumbers
```

Comment:
What is the difference between that and `type="number"`?

Reply:
Input with `type="number"` doesn't allow to type dots